### ISSUE TITLE
Editor menu dropdowns are missing a11y role attributes

### ISSUE THREAD

**Issue by gcasar** [StackExchange/Stacks-Editor#245]
When using the editor and clicking on a editor button "More formatting" a popover appears that correctly has a `role="menu"` attribute specified. The child items are missing their own `role="menuitem"` attributes.

**To Reproduce**
1. Go to Ask a question page
2. Expand the (...) dropdown "More formatting"
3. Inspect attributes of the button element

**Expected behavior**
The popover should follow [ARIA menu role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/menu_role)

**Screenshots**
![image](https://user-images.githubusercontent.com/2477157/236774038-11873829-5938-42f9-a8de-f0a72c87b7c4.png)

**Desktop (please complete the following information):**
 - OS: Windows 
 - Browser: Edge
 - Version 113
 
 The best starting place for a possible fix if I were to try and tackle this: https://github.com/StackExchange/Stacks-Editor/blob/2124fd6ec1fb0b6569a4f9d2fea8686984120292/src/shared/menu/helpers.ts#L192-L203

**Comment by giamir** [comment]
This was resolved in https://github.com/StackExchange/Stacks-Editor/commit/f00095e87ab39ef2b05100e98c9cb6305b9ca182
